### PR TITLE
Button

### PR DESCRIPTION
**Pull request by ckahub**
#Button
create different styles of button
#SolidButton
BUTTON01
#LinkButton
button02

#FloatingButton
button03

### CODE REVIEW

**Review by aurobindoGupta — APPROVED** [review]
looks good

File: Components/Button/button.html [hunk 1]
```diff
@@ -0,0 +1,256 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Button Component</title>
```
Comment (surbhi84) [inline Components/Button/button.html:7]: you can add meta author and description tags

File: Components/Button/button.html [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+            <div class="content_Head">
+                Link Button
+            </div>
+            <div class="content_Subtitle">
+                Want to attach a link action, at a button tap? Use Link Button
+            </div>
+            <div class="example_Div">
+                <button class="btn outline_C"><a href="../Avatar/avatar.html">AVATAR</a></button>
+                <button class="btn outline_A"><a href="../Alert/alert.html">ALERT</a></button>
+                <button class="btn outline_B"><a href="../Badges/badges.html">BADGES</a></button>
```
Comment (surbhi84) [inline Components/Button/button.html:166]: meaningful classnames should be used instead of a, b c

**Review by surbhi84 — APPROVED** [review]
Rest looks good to me.

File: Components/Button/button.html (hunk 1 above)
Comment (ckahub) [reply Components/Button/button.html:7]: ok i will do the needful

File: Components/Button/button.html (hunk 2 above)
Comment (ckahub) [reply Components/Button/button.html:166]: good suggestion

**Pull request merged**